lnutil/litadr.go

```diff
@@ -71,6 +72,13 @@ func LitAdrFromPubkey(in [33]byte) string {
 	return bech32.Encode("ln", doubleSha[:20])
 }
 
+// Given a koblitz pubkey, returns the lit addr
+func ConvertPubkeyToLitAddr(in *koblitz.PublicKey) string {
```

Comment:
We're supposed to be taking things *out* of lnutil, not putting them back in...

Reply:
`lnutil` is supposed to be a collection of such utility functions unless I'm missing something here.

Reply:
Yes but it's becoming a conglomeration of a bunch of things and it's difficult to do anything without pulling in lnutil.  To be fair this is the kind of thing that doesn't matter that much to be in there, I was more annoyed with the numerous `*lib.go` files that should be in better places.